src/components/LoansByCategory/ViewToggle.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+	<div class="view-toggle">
+		<router-link
+			to="/lend-by-category"
+			class="browse-box">
+			<span>
+				<kv-icon
+					name="browse-toggle"
+					class="browse-toggle-icon"/>
+				<span class="small-text hide-for-small-only view-text">Explore</span>
+			</span>
+		</router-link>
+		<router-link
+			to="/lend"
+			class="filter-box">
+			<span class="divider">
+				<kv-icon
+					name="filters-toggle"
+					class="filter-toggle-icon"/>
+				<span class="small-text hide-for-small-only view-text">Filter</span>
+			</span>
+		</router-link>
+	</div>
+</template>
```

Comment:
@BoulderBrains Need to add in the analytics for these links, basic reqs are in 473...
['Lending', 'click-filter-toggle', ##link name##]

Reply:
I'm going to use upper case for my `link name`: `Browse`

Reply:
FYI: I went with the following line: 
For the explore/browse icon:
v-kv-track-event="['Lending', 'click-filter-toggle', 'Explore']"
For the filter icon: 
v-kv-track-event="['Lending', 'click-filter-toggle', 'Filter']"